Libraries/Components/Button.windows.js

```diff
@@ -0,0 +1,191 @@
+     */
+    color: ColorPropType,
+    /**
+     * If true, disable all interactions for this component.
+     */
+    disabled: React.PropTypes.bool,
+    /**
+     * Handler to be called when the user taps the button
+     */
+    onPress: React.PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {
+      accessibilityLabel,
+      color,
+      onPress,
+      title,
+      disabled,
+    } = this.props;
+    const buttonStyles = [styles.button];
+    const textStyles = [styles.text];
+    const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    if (color && Platform.OS === 'ios') {
+      textStyles.push({color: color});
+    } else if (color && Platform.OS === 'windows') {
+      textStyles.push({color: color});      
+    } else if (color) {
+      buttonStyles.push({backgroundColor: color});
+    }
+    if (disabled) {
+      buttonStyles.push(styles.buttonDisabled);
+      textStyles.push(styles.textDisabled);
+    }
+    invariant(
+      typeof title === 'string',
+      'The title prop of a Button must be a string',
+    );
+    const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
+    return (
+      <Touchable
+        accessibilityComponentType="button"
+        accessibilityLabel={accessibilityLabel}
+        accessibilityTraits={['button']}
+        disabled={disabled}
+        onPress={onPress}>
+        <View style={buttonStyles}>
+          <Text style={textStyles}>{formattedTitle}</Text>
+        </View>
+      </Touchable>
+    );
+  }
+}
+
+// Material design blue from https://material.google.com/style/color.html#color-color-palette
+let defaultBlue = '#2196F3';
+if (Platform.OS === 'ios') {
+  // Measured default tintColor from iOS 10
+  defaultBlue = '#0C42FD';
+} else if (Platform.OS === 'windows') {
+  // Measured default tintColor from iOS 10
+  defaultBlue = '#0C42FD';
+}
+
+const styles = StyleSheet.create({
+  button: Platform.select({
+    ios: {},
+    android: {
+      elevation: 4,
+      backgroundColor: defaultBlue,
+      borderRadius: 2,
+    },
+    windows: {
+      backgroundColor: 0x00000033,
+      paddingLeft: 8,
+      paddingTop: 4,
+      paddingRight: 8,
+      paddingBottom: 4,
+    }
+  }),
+  text: Platform.select({
+    ios: {
+      color: defaultBlue,
+      textAlign: 'center',
+      padding: 8,
+      fontSize: 18,
+    },
+    android: {
+      textAlign: 'center',
+      color: 'white',
+      padding: 8,
+      fontWeight: '500',
+    },
+    windows: {
+      color: 'black',
+      textAlign: 'center',
+      fontSize: 15,
+    }
+  }),
+  buttonDisabled: Platform.select({
+    ios: {},
+    android: {
+      elevation: 0,
+      backgroundColor: '#dfdfdf',
+    },
+    windows: {}
+  }),
+  textDisabled: Platform.select({
+    ios: {
+      color: '#cdcdcd',
+    },
+    android: {
+      color: '#a1a1a1',
+    },
+    windows: {
+      color: 0x00000066,
```

Comment:
@kevinvangelder can you give an example of the correct way to specify this value?